src/lib/y2storage/dialogs/guided_setup/select_scheme.rb

```diff
@@ -24,15 +24,20 @@
 require "y2storage/dialogs/guided_setup/base"
 
 Yast.import "InstExtensionImage"
+Yast.import "Popup"
 
 module Y2Storage
   module Dialogs
     class GuidedSetup
       # Dialog to select partitioning scheme.
       class SelectScheme < Base
-        def encryption_handler
-          widget_update(:password, widget_value(:encryption), attr: :Enabled)
-          widget_update(:repeat_password, widget_value(:encryption), attr: :Enabled)
+        # Handler for :encryption check box.
+        # @param allow_focus [Boolean] whether password field can be focused
+        def encryption_handler(allow_focus: true)
```

Comment:
Umm, I think we are not understanding the same. The field for password can have three situations:
* (a) not enabled and not focused
* (b) enabled and focused
* (d) enabled and not focused

States (a) and (b) occur when encryption checkbox is checked/unchecked, and state (c) could happen when dialog is opened and a password exists. In this case we do not want focus the field, but it is enabled. So, the param :allow_focus is for controlling this case.